asyncpg/connection.py

```diff
@@ -909,6 +930,25 @@ def _cancel_current_command(self, waiter):
 
         self._loop.create_task(cancel())
 
+    def _notice(self, message):
+        # `self._notice_cb` is checked internally and passed here as the `cb`
```

Comment:
Add a 'period' at the end of comment sentences. Also the comment looks outdated.